src/services/resources-getter.js

```diff
@@ -17,12 +17,14 @@ function ResourcesGetter(model, options, params) {
   const OPERATORS = new Operators(options);
   const primaryKey = _.keys(model.primaryKeys)[0];
   const filterParser = new FiltersParser(schema, params.timezone, options);
+  let fieldNamesRequested;
+  let searchBuilder;
```

Comment:
It seems there are more changes that just "Filter parser is now asynchronous" in this PR (+36/-24 in this `resources-getter` file). 

Could you also add a note in this ticket to explain why this file changed too? (it seems a function was renamed and its behavior as changed). Maybe you could update the changelog too? (or extract this part in one other PR if possible).

Reply:
OK, thank you!